netcore/System.Private.CoreLib/src/System/GC.cs

```diff
@@ -236,10 +237,29 @@ internal static ulong GetSegmentSize ()
 			return 1024 * 1024 * 16;
 		}
 
+
+		[MethodImplAttribute (MethodImplOptions.InternalCall)]
+		private static extern GCMemoryInfo _GetGCMemoryInfo (ref long highMemoryLoadThresholdBytes,
+														     ref long memoryLoadBytes,
+															 ref long totalAvailableMemoryBytes,
+															 ref long heapSizeBytes,
+															 ref long fragmentedBytes); 
+
 		public static GCMemoryInfo GetGCMemoryInfo ()
 		{
-			// TODO: Implement
-			return default;
+			long highMemoryLoadThresholdBytes = 0L;
+			long memoryLoadBytes = 0L;
```

Comment:
No need for 'L's.
The parameters can be out long, so the variables don't need to be initialized.

Reply:
`out` params worry me, as they imply a burden on the callee that is often not me.
`ref` params feel safer to me.